health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/Response.js

```
@@ -2,10 +2,10 @@ import React, { useState, Fragment } from "react";
 import { Link, useHistory, useLocation } from "react-router-dom";
 import { useTranslation } from "react-i18next";
 import { ActionBar, SubmitBar, ArrowLeft, ArrowForward } from "@egovernments/digit-ui-react-components";
+import { Button } from "@egovernments/digit-ui-components";
 import { PanelCard } from "@egovernments/digit-ui-components";
 import { Button } from "@egovernments/digit-ui-react-components";
```

Biome

[error] Shouldn't redeclare 'Button'. Consider to delete it or rename it.

'Button' is defined here:


(lint/suspicious/noRedeclare)